thankyou.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Thank you for subscribing</title>
+
+    <meta name="description" content="Decentraland is a VR world that uses a blockchain to determine ownership of land, and its own coin to trade land and purchase goods and services inside the world.">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="robots" content="noindex,nofollow">
+
+    <link rel="shortcut icon" href="favicon.ico" />
+    <link rel="canonical" href="https://decentraland.org" />
+
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/main.min.css">
+  </head>
+  <body class="thankyou">
+    <header class="container-fluid">
+      <div class="navbar" role="navigation">
+        <div class="navbar-header">
+          <a href="/" class="navbar-logo">
+            <i class="icon icon-decentraland pull-left"></i>
+            <h1 class="pull-left">Decentraland</h1>
+          </a>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-xs-12 text-center">
+          <h2 class="title"><b>Thank you</b> for subscribing to our newsletter!</h2>
+          <div class="description">
+            <p>Almost finished...we sent you a confirmation email to complete the subscription process.</p>
+            <br />
+            <p>We're extra careful with the email volume. If you want to read something right now, head over to <a href="https://slack.decentraland.org" target="_blank">our Slack channel</a> or <a href="https://medium.com/decentraland">blog</a>.</p>
+          </div>
```

Comment:
*Thank you for subscribing!*

Newsletter subscribers receive important updates and project milestones first. We sent you an email to confirm your access.

We only email major announcements and developments, so for complete access, join our Slack and Telegram channels or explore our blog.